NavigationReactNative/src/TitleBarIOS.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import { requireNativeComponent, Platform, StyleSheet } from 'react-native';
+
+const NVTitleBarView = requireNativeComponent<any>('NVTitleBarView', null)
```

Comment:
Thanks for doing the rename but this should be called `NVTitleBar` instead of `NVTitleBarView`